src/app/screens/LNURLPay/index.tsx

```diff
@@ -314,10 +314,10 @@ function LNURLPay() {
                   <dl>
                     <>
                       {formattedMetadata(details.metadata).map(([dt, dd]) => (
-                        <>
+                        <div key={dt}>
```

Comment:
~Hm, issue with this is, that this is not valid html anymore.~
~Within `dl` only `dt`/`dd` can exist.~
That was what I wanted to write, but turns out:
<img width="507" alt="image" src="https://user-images.githubusercontent.com/1016218/184465184-678d28d9-bde6-4f5f-b062-b85e3343b63a.png">
(via https://validator.w3.org/nu/#textarea)
Weird, but ok. A `div` within a `ul` is not valid. Thought it will be the same issue within a `dl`.

But only using `dt` as key could be an issue maybe. If i.e. for some reason a `dt` could be not unique. Maybe this should be combined with an `index`?

Reply:
Done! Changed it to use Fragment and use index for key.